model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -14,30 +15,65 @@
     ["IHR", "predicted"],
     defaults=[None, None],
 )
-"""Output from HospitalAdmissions.sample()"""
+HospAdmissionsSample.__doc__ = """
+A container for holding the output from HospAdmissionsSample.sample.
+
+Attributes
+----------
+IHR : float or None
+    The infected hospitalization rate. Defaults to None.
+predicted : ArrayLike or None
+    The predicted number of hospital admissions. Defaults to None.
+
+Notes
+-----
+TODO: Add Notes.
+"""
 
 InfectHospRateSample = namedtuple(
     "InfectHospRateSample",
     ["IHR"],
     defaults=[None],
 )
+InfectHospRateSample.__doc__ = """
+A container for holding the output from InfectHospRateSample.sample.
+
+Attributes
+----------
+IHR : ArrayLike or None
+    The infected hospitalization rate. Defaults to None.
+
+Notes
+-----
+TODO: Add Notes.
+"""
 
 
 class InfectHospRate(RandomVariable):
-    """Infection to Hospitalization Rate"""
+    """
+    Infection to Hospitalization Rate
+
+    Methods
+    -------
+    validate(distr)
+        Validates distribution is Numpyro distribution
+    sample(**kwargs)
+        Produces a sample of the IHR
+    """
 
     def __init__(
         self,
-        dist: dist.Distribution,
-        varname: str = "IHR",
+        dist: Optional[dist.Distribution] = dist.LogNormal(jnp.log(0.05), 0.05),
```

Comment:
```suggestion
        dist: dist.Distribution | None = dist.LogNormal(jnp.log(0.05), 0.05),
```
This is not a necessary change, just a suggestion. In 3.10+, you can use this notation instead of `Optional[..]`. I personally find it slightly nicer to read, but it's really not of much consequence. See [docs here](https://docs.python.org/3/library/typing.html#typing.Optional)

Reply:
`dist` should default to `None`, not to a specific instantiated distribution (the latter was an artifact of old example code)
``` suggestion
        dist: dist.Distribution | None = None
```

Reply:
Addressed in 0710c3a